### PR TITLE
Make composer send on ctrl+enter

### PR DESCRIPTION
**Pull request by RoiEXLab**

### CODE REVIEW

**Comment by RoiEXLab** [comment]
@julianlam @barisusakli This PR is now open for half a year ^^

File: static/lib/composer.js [hunk 1]
```diff
@@ -311,6 +311,16 @@ define('composer', [
 			$(this).attr('disabled', true);
 			post(post_uuid);
 		});
+		
+		postContainer.keypress(function (event) {                                 
+			var keyCode = (event.which ? event.which : event.keyCode);
+			if (keyCode === 10 || keyCode == 13 && event.ctrlKey) {
```
Comment (barisusakli) [inline static/lib/composer.js]: This should be `if ((keyCode === 10 || keyCode == 13) && event.ctrlKey) {`

Comment (barisusakli) [inline static/lib/composer.js]: I think this should be `if ((keyCode === 10 || keyCode == 13) && event.ctrlKey) {` as per https://stackoverflow.com/a/9343095/583363

Comment (RoiEXLab) [reply static/lib/composer.js]: Indeed @barisusakli

**Comment by RoiEXLab** [comment]
@barisusakli Updated! Ready for re-review

**Pull request merged**